mailing_list/models.py

```diff
@@ -111,9 +119,37 @@ def __unicode__(self):
             self.section_id
         )
 
+    def _get_unsubscribed_email_set(self):
+        return {x.email for x in self.unsubscribed_set.all()}
```

Comment:
I don't see where `unsubscribed_set` is defined -- am I missing something? Does it get generated automatically?